src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -222,6 +222,72 @@ export default defineConfig({
 });
 ```
 
+### isr
+
+**Type:** `boolean | VercelISRConfig`<br/>
+**Available for:** Serverless
+<Since v="7.2.0" pkg="@astrojs/vercel" />
+**Default**: 'false'
+
+Allows your project to be deployed as an [ISR (Incremental Static Regeneration)](https://vercel.com/docs/incremental-static-regeneration) function, which caches your on-demand rendered pages in the same way as prerendered pages after first request. Note that ISR function requests do not include search params, similar to [requests](/en/reference/api-reference/#astrorequest) in static mode.
+
+```js title="astro.config.mjs" ins={8}
+import { defineConfig } from 'astro/config';
+import vercel from '@astrojs/vercel/serverless';
+
+export default defineConfig({
+  // ...
+  output: 'server',
+  adapter: vercel({
+    isr: true,
+  }),
+});
+```
+
+### VercelISRConfig
+
+By default, an ISR function caches for the duration of your deployment. You can further control caching using `bypassToken`, `exclude`, and `expiration`.
+
+```js title="astro.config.mjs" ins={8}
+import { defineConfig } from 'astro/config';
+import vercel from '@astrojs/vercel/serverless';
+
+export default defineConfig({
+  // ...
+  output: 'server',
+  adapter: vercel({
+    isr: {
+      /* 
+       * A secret random string that you create.
+       * 
+       * Its presence in the `__prerender_bypass` cookie will result in fresh responses being served, bypassing the cache. See Vercel’s documentation on [Draft Mode](https://vercel.com/docs/build-output-api/v3/features#draft-mode) for more information.
+       * Its presence in the `x-prerender-revalidate` header will result in a fresh response which will then be cached for all future requests to be used. See Vercel’s documentation on [On-Demand Incremental Static Regeneration (ISR)](https://vercel.com/docs/build-output-api/v3/features#on-demand-incremental-static-regeneration-isr) for more information.
+       * 
+       * @default `undefined`
+       */
+      bypassToken: process.env.BYPASS_TOKEN,
+      
+      /**
+       * Paths that will always be served by a serverless function instead of an ISR function.
+       * 
+       * @default `[]`
+       */
+    	exclude: [ "/api/invalidate" ],
+
+      /**
+       * Expiration time (in seconds) before the pages will be re-generated.
+       * 
+       * Setting to `false` means that the page will stay cached as long as the current deployment is in production.
+       * 
+       * @default `false`
+       */
+      expiration: 60 * 60 * 24;
```

Comment:
JSDoc comments like this are not as accessible or discoverable as regular paragraphs. Any chance we could split them into subheadings with this content?

Reply:
Yeah, it's a bit much.  I'll look into it.

Reply:
Also noting that Algolia does not index text inside code samples, so no real important explanations/descriptions/definitions should ever *only* exist inside code snippets!

Code comments are best as as single one-line *reminders* (or draw attention to-ers) of things previously described in text.